soco/core.py

```diff
@@ -1255,6 +1266,34 @@ def get_sonos_playlists(self, *args, **kwargs):
         args = tuple(['sonos_playlists'] + list(args))
         return self.get_music_library_information(*args, **kwargs)
 
+    def get_sonos_playlist_by_attr(self, attr_name, match):
+        """ Return the first Sonos Playlist DidlPlaylistContainer that
+        matches the attribute specified.
+
+        Args:
+            attr_name (str): DidlPlaylistContainer attribute to compare.
+            match (str): Value to match.
+
+        Returns:
+            (:py:class:`~.soco.data_structures.DidlPlaylistContainer`): The
```

Comment:
I don't think this changes with Napoleop. Napoleon is mostly about formatting the list of Args, Returns, and Raises and their type.